### PR TITLE
Add note that nest.help('model_name') only works if the documentation is built

### LINKED ISSUE #2483: No `nest.help()` when installed through Conda
The NEST help doesn't work when installed through conda. So `nest.help('iaf_psc_exp')` gives the following error:

    Sorry, there is no help for model 'iaf_psc_exp'.
    Use the Python help() function to obtain help on PyNEST functions.

This should either give useful information, or redirect to online documentation.

**Supporting materials**
This function is mentioned in different workshop and tutorial materials.

**Desktop/Environment**
Various, since it's in a workshop environment potentially with own systems.

### PR DESCRIPTION
**Pull request by steffengraber**
Fixes #2483.
The problem that no documentation is created in the conda package still exists. However, the user receives a short note about this.

### CODE REVIEW

**Comment by steffengraber** [comment]
After discussions in the NEST Developer and NEST Documentation meeting it was decided to remove the reference to `nest.help('model_name')` and instead refer to the online documentation.

File: doc/htmldoc/getting_help.rst [hunk 1]
```diff
@@ -33,9 +33,5 @@ Model information
...
    nest.synapse_models
 
-* To get details on model equations and parameters, use:
-
-.. code-block:: python
-
-   nest.help('model_name')
-
+* To get details on the model equations and parameters,
+  please check our documentation at https://nest-simulator.readthedocs.io/.
```
Comment (jessica-mitchell) [inline doc/htmldoc/getting_help.rst:37]:
```suggestion
  please check out our `online model documentation <https://nest-simulator.readthedocs.io/en/stable/models/index.html>`_.
```

File: pynest/nest/__init__.py [hunk 2]
```diff
@@ -29,8 +29,8 @@
 
 * ``nest.synapse_models`` shows all available synapse models.
 
-* ``nest.help("model_name") displays help for the given model, e.g.,
-  ``nest.help("iaf_psc_exp")``
+* To get details on the model equations and parameters,
+  please check our documentation at https://nest-simulator.readthedocs.io/.
```
Comment (jessica-mitchell) [inline pynest/nest/__init__.py:33]:
```suggestion
  please check out our model documentation at https://nest-simulator.readthedocs.io/en/stable/models/index.html.
```

File: doc/htmldoc/getting_help.rst (hunk 1 above)
Comment (steffengraber) [reply doc/htmldoc/getting_help.rst:37]: Thank you for your input. The path to the model documentation could change over time. We would then have to check the path with every release.  I avoid that with the general link. Or do you think that's not a problem?

**Review by jessica-mitchell — APPROVED** [review]
thanks!

**Comment by jessica-mitchell** [comment]
Merging with 1 reviewer because documentation PR

**Pull request merged**